### PR TITLE
fix: Adjust artwork rail height when font scaling is enabled

### PR DESCRIPTION
**Pull request by olerichter00**
### Description

This PR adds a font-scaling factor to the artwork rail text height to avoid the text being cut off.

| Before | After |
| --- | --- |
|Simulator Screen Shot - iPhone 13 Pro - 2022-10-05 at 15 02 58 | Simulator Screen Shot - iPhone 13 Pro - 2022-10-05 at 14 57 02|
| --- | --- |
|Simulator Screen Shot - iPhone 13 Pro - 2022-10-05 at 14 50 22 |Simulator Screen Shot - iPhone 13 Pro - 2022-10-05 at 15 02 34 |

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Components/ArtworkRail/ArtworkRailCard.tsx [hunk 1]
```diff
@@ -47,6 +47,8 @@ export const ArtworkRailCard: React.FC<ArtworkRailCardProps> = ({
   testID,
   ...restProps
 }) => {
+  const fontScale = PixelRatio.getFontScale()
```
Comment (gkartalis) [inline src/app/Components/ArtworkRail/ArtworkRailCard.tsx:50]: **praise:** ooooh nice didn't know about this one👌👌👌

File: .secrets.baseline [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.3.0",
+  "version": "1.4.0",
```
Comment (gkartalis) [inline .secrets.baseline:2]: 👀 how did this happen?

**Review by gkartalis — APPROVED** [review]

File: .secrets.baseline (hunk 2 above)
Comment (olerichter00) [reply .secrets.baseline:2]: Not sure 🤔 I'll update the PR to revert the version update and try to fix the version locally.

**Review by gkartalis — APPROVED** [review]

**Comment by gkartalis** [comment]
Screenshot 2022-10-05 at 15 25 05
👀 is the ci broken? 🤯

**Comment by pvinis** [comment]
assigning back to you @olerichter00. merge when you want.

**Pull request merged**